routes.js

```diff
@@ -1,58 +1,79 @@
-const { error } = require('console');
 const express = require('express');
-const router = express.Router();
+const createMyCustomApi = express.Router();
+const fs = require('fs');
 
-// Middleware
-router.use(express.json());
+// Fetch data from the members.json file
+const loadMembers = () => {
+    const jsonData = fs.readFileSync(__dirname + "/" + "members.json");
+        return JSON.parse(jsonData);
+}
 
-// Get method (Get all member)
-router.get('/getAllMembers', (req, res) => {
-  res.send(teamMembersArray);
-  if(!error) {
-      res.status(200);
-  }
-});
+// Update members.js file
+const saveMembers = (newData) => {
+    const stringifyData = JSON.stringify(newData);
+    fs.writeFileSync(__dirname + "/" + "members.json", stringifyData)
+}
 
-// Post method (Add a new member)
-router.post('/addNewMember', (req, res) => {
-  const member = {
-    memberId: req.body.memberId,
-    role: req.body.role,
-    firstname: req.body.firstname,
-    lastname: req.body.lastname
-  }
-  teamMembersArray.push(member);
-  res.send(member);
-  if(!error) {
-    res.status(200);
-  }
-});
+// Endpoint to get a list of all team members (get request)
+createMyCustomApi.get('/getAllMembers', function(req, res) {
+    fs.readFile(__dirname + "/" + "members.json", 'utf8', function(err, data) {
```

Comment:
Could you instead reuse you loadMemebers function here?

Reply:
Done!